### PR TITLE
Update doc on minimum required Gradle version

### LINKED ISSUE #41: Update docs to require Gradle 4.9
### Expected behavior
Expect this to work with Gradle 4.3

### Actual behavior
```
Creating deblibs tasks...


FAILURE: Build failed with an exception.

* What went wrong:
org.gradle.api.tasks.TaskContainer.register(Ljava/lang/String;Ljava/lang/Class;Lorg/gradle/api/Action;)Lorg/gradle/api/tasks/TaskProvider;
```

### Steps to reproduce the behavior
 - Install gradle 4.3
 - run `./gradlew --stacktrace --info createSlackMessage`

From the gradle docs, it looks like this endpoint was added in 4.9:
https://docs.gradle.org/current/javadoc/org/gradle/api/tasks/TaskContainer.html#register-java.lang.String-org.gradle.api.Action-

### PR DESCRIPTION
**Pull request by eyedol**
Fixes #41 

This `PR` makes the following changes:

- Upgrades dependencies
- Update doc to reflect required Gradle version

### CODE REVIEW

File: src/main/kotlin/com/hellofresh/deblibs/slack/SlackMessage.kt [hunk 1]
```diff
@@ -36,7 +36,7 @@ data class SlackMessage(
         enum class Color(val hexColor: String) {
             RED("#c4291c"),
             GREEN("#36a64f"),
-            YELLOW("#d6a048")
+            YELLOW("#d6a048"),
```
Comment (siempredelao) [inline src/main/kotlin/com/hellofresh/deblibs/slack/SlackMessage.kt]: Is this last comma valid? I mean, doesn't it throw an error at compile time?
```suggestion
            YELLOW("#d6a048")
```

Comment (eyedol) [reply src/main/kotlin/com/hellofresh/deblibs/slack/SlackMessage.kt]: Oh, it's valid. It compiles. I read somewhere it's a good thing to do as future proof. When we add another a new enum it doesn't affect changes

Comment (siempredelao) [reply src/main/kotlin/com/hellofresh/deblibs/slack/SlackMessage.kt]: Alright, thanks for the feedback. I'm approving the PR :smiley:

**Comment by siempredelao** [comment]
+1

File: src/main/kotlin/com/hellofresh/deblibs/slack/SlackMessage.kt (hunk 1 above)
Comment (eyedol) [reply src/main/kotlin/com/hellofresh/deblibs/slack/SlackMessage.kt]: No problem. Thanks for the review.

**Comment by lobo-hellofresh** [comment]
+1

**Pull request merged**